sphinx_gallery/binder.py

```diff
@@ -103,12 +111,26 @@ def gen_binder_rst(fpath, binder_conf, gallery_conf):
     binder_conf = check_binder_conf(binder_conf)
     binder_url = gen_binder_url(fpath, binder_conf, gallery_conf)
 
+    binder_logo = os.path.join(glr_path_static(), 'binder_badge_logo.svg')
+    binder_logo_full_path = os.path.join(
+        target_dir, 'images', 'binder_badge_logo.svg'
+    )
+    if gallery_conf['gallery_dirs'] is None:
+        # Avoid copying file when testing
+        pass
+    else:
+        if not os.path.exists(binder_logo_full_path):
+            shutil.copyfile(binder_logo, binder_logo_full_path)
+    sources_dir = gallery_conf['src_dir']
+    binder_logo_rel_path = os.path.relpath(binder_logo_full_path, sources_dir)
+    binder_logo_rel_path = binder_logo_rel_path.replace(os.sep, '/')
+
     rst = (
         "\n"
         "  .. container:: binder-badge\n\n"
-        "    .. image:: https://mybinder.org/badge_logo.svg\n"
+        "    .. image:: /{}\n"
```

Comment:
I copied this from:
https://github.com/sphinx-gallery/sphinx-gallery/blob/b09770c9dae3eeb8c10b5e45b70fd19f9db70cd1/sphinx_gallery/scrapers.py#L429

but I don't understand how the `/` works for windows OS?

Reply:
I think it should be fine. IIRC sphinx makes it relative to the build root, so I'd expect something like `/_static/binder_badge_logo.svg` could be used directly